src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -159,12 +160,21 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             // publish the place after creating it.
             // Existing places should always keep their original status, so
             // only do this publish command for new places.
-            $placeAggregate->publish($publishDate);
+            // For now the trigger to do a publish is the imports path.
+            // In the future this needs to be fine-tuned with either default values
+            // or config for specific API partners.
```

Comment:
Later it would be defined by an explicit `workflowStatus` property in the JSON. And if the property is not there, we would still check the path to decide the default (publish or not) for backward compatibility with existing integrators.

Reply:
I added a reference to ticket https://jira.uitdatabank.be/browse/III-4609